java/src/main/java/sh/platform/languages/sample/MongoDBSample.java

```diff
@@ -0,0 +1,39 @@
+package sh.platform.languages.sample;
+
+import com.mongodb.MongoClient;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import org.bson.Document;
+import sh.platform.config.Config;
+import sh.platform.config.MongoDB;
+
+import java.util.function.Supplier;
+
+import static com.mongodb.client.model.Filters.eq;
+
+public class MongoDBSample implements Supplier<String> {
```

Comment:
Is the Supplier interface still needed?  It's just a named method returning a string, no?

Reply:
This supplier avoid to use Reflection, makes the code cleaner. Also, it comes from Java 8, thus it is fine.